migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/table.rs

```diff
@@ -173,5 +173,5 @@ fn indexes_match(first: &IndexWalker<'_>, second: &IndexWalker<'_>) -> bool {
             names_match && lengths_match && orders_match
         })
         && first.index_type() == second.index_type()
-        && first.algorithm() == second.algorithm()
+        && first.algorithm().unwrap_or_default() == second.algorithm().unwrap_or_default()
```

Comment:
This is still error prone (you would have to do that at every call site) — would it be possible for the algorithm field not to be an `Option`?

Reply:
We decided to go with the unwrap for now and take a look at the optionality separately.